UnitTests/GitUI.Tests/Script/ScriptOptionsParserTests.cs

```diff
@@ -362,7 +362,7 @@ public void ParseScriptArguments_resolve_HEAD_with_checkedout_branch_name()
             _module.GetSelectedBranch(emptyIfDetached: true).Returns(branchName);
 
             string result = ScriptOptionsParser.GetTestAccessor().ParseScriptArguments(
-                arguments: "{" + option + "}", option,
+                arguments: "{" + option + "}", option: option,
```

Comment:
If the compiler does not force us to add this, we should not add that redundancy.